docs/standard/exceptions/handling-com-interop-exceptions.md

```diff
@@ -22,6 +22,13 @@ Managed and unmanaged code can work together to handle exceptions. If a method t
   
  If a method fails in unmanaged code, an exception can be passed to a managed code segment. The topic [HRESULTS and Exceptions](../../../docs/framework/interop/how-to-map-hresults-and-exceptions.md) contains a table showing how HRESULTS map to runtime exception objects.  
 
+## Longjump Behaviors
+Longjump interop is held to the same constraints as exception handling and is currently only supported on Windows.
+
+When unmanaged code moves on through a `longjmp` instruction, the runtime observes different behaviors between Windows and Unix operating systems. On Windows, there is support to unwind the stack properly when dealing with exceptions thrown by external components. On the other hand, Unix does not support exception interop as currently the Unix ABI has no definition for it. Therefore, exceptions thrown this way end up resulting in unpredictable behaviors and potential crashes.
```

Comment:
So, taking into account your thoughts from your last comment, we could draft the paragraphs like this:

- The first paragraph should only refer to specifying C++ exception interop is not officially supported due to the portability issues that arise since managed code can't know how the C++ exception mechanisms work under the covers.
- The second paragraph explains the `longjmp` case.

What do you think? How can we improve this?

Reply:
> C++ exception interop is not officially supported due to the portability issues

... on non-Windows platforms.

> officially supported

For the docs - omit "officially". If something is mentioned in the docs as supported, it implies that it is supported officially.

Reply:
Looks good otherwise.

You can also reference https://www.mono-project.com/docs/advanced/pinvoke/#runtime-exception-propagation (e.g. as a link at the end - check existing topics for how it is done) or borrow some of the ideas for how to explain this from it.